src/main/kotlin/it/pagopa/ecommerce/payment/requests/services/PaymentRequestsService.kt

```diff
@@ -92,116 +78,50 @@ class PaymentRequestsService(
-          verifyPaymentNoticeReq.qrCode = qrCode
-          paymentRequestInfo =
-            nodeForPspClient
-              .verifyPaymentNotice(
-                objectFactoryNodeForPsp.createVerifyPaymentNoticeReq(verifyPaymentNoticeReq))
-              .flatMap { verifyPaymentNoticeResponse ->
-                val isNodoError = isNodoError(verifyPaymentNoticeResponse)
-                logger.info(
-                  "VerifyPaymentNotice: outcome: [{}] fault code: [{}] is nodo error: [{}]",
-                  verifyPaymentNoticeResponse.outcome,
-                  verifyPaymentNoticeResponse?.fault?.faultCode,
-                  isNodoError)
-                if (isNodoError) {
-                  Mono.error(NodoErrorException(verifyPaymentNoticeResponse.fault))
-                } else {
-                  Mono.just(
-                    PaymentRequestInfo(
-                      id = rptId,
-                      paFiscalCode = verifyPaymentNoticeResponse.fiscalCodePA,
-                      paName = verifyPaymentNoticeResponse.companyName,
-                      description = verifyPaymentNoticeResponse.paymentDescription,
-                      amount =
-                        nodoOperations.getEuroCentsFromNodoAmount(
-                          verifyPaymentNoticeResponse.paymentList.paymentOptionDescription[0]
-                            .amount),
-                      dueDate =
-                        getDueDateString(
-                          verifyPaymentNoticeResponse.paymentList.paymentOptionDescription[0]
-                            .dueDate),
-                      isNM3 = true,
-                      paymentToken = null,
-                      idempotencyKey = null,
-                      isCart = false))
-                }
-              }
-        } else {
-          val enteBeneficiario: CtEnteBeneficiario? =
-            esitoNodoVerificaRPTRisposta.datiPagamentoPA?.enteBeneficiario
-          paymentRequestInfo =
-            Mono.just(
-              PaymentRequestInfo(
-                id = rptId,
-                paFiscalCode =
-                  enteBeneficiario?.identificativoUnivocoBeneficiario?.codiceIdentificativoUnivoco,
-                paName = enteBeneficiario?.denominazioneBeneficiario,
-                description = esitoNodoVerificaRPTRisposta.datiPagamentoPA.causaleVersamento,
-                amount =
-                  nodoOperations.getEuroCentsFromNodoAmount(
-                    esitoNodoVerificaRPTRisposta.datiPagamentoPA.importoSingoloVersamento),
-                dueDate = null,
-                isNM3 = false,
-                paymentToken = null,
-                idempotencyKey = null,
-                isCart = false))
-        }
-        return@flatMap paymentRequestInfo
-      }
+    Mono.just(rptId).flatMap { request: RptId ->
+      logger.info(
+        "Calling Nodo for VerifyPaymentNotice for get payment info for rptId: [{}]. PaymentContextCode: [{}]",
+        rptId.value,
```

Comment:
use here the `request` variable from line 81 or remove` request: RptId` altogether

Reply:
Fixed !! 😎